Cargo.toml

```diff
@@ -132,3 +116,18 @@ tracing-subscriber = { version = "0.3", default-features = false, features = [
 url = "2.5"
 uuid = { version = "1.10", default-features = false }
 wildmatch = "2.3"
+
+snops-checkpoint = { path = "./crates/snops-checkpoint" }
+snops-common = { path = "./crates/snops-common" }
+
+# snarkos-account = { path = "../snarkos/account" }
```

Comment:
is it possible to feature gate these local path ones or does that completely break cargo